### PR TITLE
Segment security profile resource

### PR DESCRIPTION
**Pull request by annakhm**

### CODE REVIEW

File: nsxt/resource_nsxt_policy_segment_security_profile.go [hunk 1]
```diff
@@ -0,0 +1,317 @@
...
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  true,
+			},
+			"dhcp_server_block_v6_enabled": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  true,
+			},
+			"non_ip_traffic_block_enabled": {
```
Comment (ksamoray) [inline nsxt/resource_nsxt_policy_segment_security_profile.go:69]: Aren't we usually adding description to resource attributes?

Comment (annakhm) [reply nsxt/resource_nsxt_policy_segment_security_profile.go:69]: This file was generated by policygen, and descriptions in spec are often too long to appear here in the code. We do have them in documentation. I'm not sure description in the code has any practical purpose.

Comment (ksamoray) [reply nsxt/resource_nsxt_policy_segment_security_profile.go:69]: Makes sense. I wonder how much the provider image will shrink if we remove all these strings :)

**Review by ksamoray — APPROVED** [review]
LGTM

**Pull request merged**